examples/ModuleImport/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+facelift_add_interface(MainExample
+    INTERFACE_DEFINITION_FOLDER ${CMAKE_CURRENT_SOURCE_DIR}/interface
+    IMPORT_FOLDERS ${CMAKE_CURRENT_SOURCE_DIR}/interface)
+
+facelift_add_qml_plugin(ModuleImportPlugin URI mainexample
+   SOURCES ModuleImportPlugin.cpp
```

Comment:
Either indent with 4 spaces (as above) or align with ModuleImportPlugin (then do the same in facelift_add_interface above - align with MainExample there)